extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmConfig.java

```diff
@@ -53,6 +54,13 @@ public interface HibernateOrmConfig {
     @ConfigDocSection
     HibernateOrmConfigDatabase database();
 
+    /**
+     * JSON/XML mapping related configuration.
+     */
+    @Deprecated(since = "3.24", forRemoval = true)
```

Comment:
I thought the conclusion was "people will need this, so it's technically not deprecated yet"? Or did you disagree?

Reply:
ohh missed this one, thanks for noticing it, 🙈 😔 

> Or did you disagree?

No, me? nooooooo 😃